terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash

```diff
@@ -4,11 +4,14 @@
 # set -x
 exec > >(tee /tmp/user-data.log|logger -t user-data ) 2>&1
 
-export INFRASTRUCTURE_ANSIBLE_VAULT_PASSWORD=`aws --region ${region} secretsmanager get-secret-value --secret-id ansible_vault_password --output text --query 'SecretString'`
-
 git clone -b ${bootstrap_version} --single-branch https://github.com/aeternity/infrastructure.git /infrastructure
 cd /infrastructure/ansible
 
+# Authenticate the instance to CSM
+PKCS7=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/pkcs7 | tr -d '\n')
```

Comment:
Not sure on hardcoded IP.

Reply:
It's special  [AWS metadata service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-data-retrieval)

Reply:
> Not sure who used to use INFRASTRUCTURE_ANSIBLE_VAULT_PASSWORD and why not defined any longer.

It was used by Ansible playbooks, it's using Vault now.